guides/common/modules/proc_updating-operating-system.adoc

```diff
@@ -0,0 +1,36 @@
+[id="updating-operating-system_{context}"]
+= Updating the operating system
+
+{Team} recommends performing operating system updates regularly, because the minor releases patch security vulnerabilities and minor issues discovered after code is released.
+
+.Prerequisites
+* Back up your {ProjectServer}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+
+include::snip_warning-maintain-config-noop.adoc[]
```

Comment:
This warning reads:

> If you customize configuration files, manually or using a tool such as Hiera, these changes are overwritten when the maintenance script runs during upgrading or updating. You can use the --noop option with the satellite-installer to test for changes. For more information, see the Red Hat Knowledgebase solution [How to use the noop option to check for changes in Satellite config files during an upgrade.](https://access.redhat.com/solutions/3351311)

The thing is, it's sorta-useless.
- The linked KCS talks about EL6 and EL7, and mentions MongoDB.
- While this would work for OS-only updates, when there are Satellite updates `--noop` would run with the wrong installer version (yet we recommend it in the "update satellite" case too)